nuxeo-services/nuxeo-platform-web-common/src/main/java/org/nuxeo/ecm/platform/ui/web/auth/service/PluggableAuthenticationService.java

```diff
@@ -193,7 +193,9 @@ public void registerContribution(Object contribution, String extensionPoint, Com
                     log.error("Unable to create session manager", e);
                 }
             } else {
-                sessionManagers.remove(smContrib.getName());
+                if (sessionManagers.containsKey(smContrib.getName())) {
+                    sessionManagers.remove(smContrib.getName());
```

Comment:
I don't understand this change, given that `sessionManagers` is a `HashMap` there is no need to test presence before removing.